apps/data-types.md

```diff
@@ -569,6 +569,13 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | taxClass | object of [taxClassInfo](page:apps-data-types#taxclassinfo) | The information on the tax class. |
 | price | object of [price](page:apps-data-types#price) | The amount of the charged tax. |
 
+## taxModel
+
+| Attribute | Type | Description |
+| - | :-: |  - |
+| model | string | The tax model of the shop. Can be *gross* or *net*. |
+| displayTaxes | boolean | Indicates if the tax model is displayed in the shop. |
```

Comment:
displayTaxes is an attribute you can find in the MBO in Settings > Tax Calculation > Tax Model and then if you select "Prices are calculated ..." -> gross prices.

Its description in the MBO is

> Display tax included in the shopping basket, e-mails, and order documents
> 
> If the "No" option is set, tax will not be shown in the shop or on invoices. Before setting this option, you should therefore check whether the tax laws in your country allow this setting.